src/ccl/charts/chart.template.html

```diff
@@ -0,0 +1,14 @@
+<div class="chart" *ngFor="let chart of chartList">
```

Comment:
Could you clarify? I'm not sure I understand the difference between the method you're explaining and what is here